project/Dependencies.scala

```diff
@@ -1,120 +1,117 @@
+  private val catsEffectV = "2.5.3" // scala-steward:off (CROM-6564)
+  private val catsV = "2.6.1"
   private val circeConfigV = "0.8.0"
-  private val circeGenericExtrasV = "0.13.0"
-  private val circeOpticsV = "0.13.0"
-  private val circeV = "0.13.0"
-  private val circeYamlV = "0.13.1"
-  private val commonsCodecV = "1.15"
-  private val commonsIoV = "2.8.0"
-  private val commonsLang3V = "3.11"
+  private val circeGenericExtrasV = "0.14.1"
+  private val circeOpticsV = "0.14.1"
+  private val circeV = "0.14.1"
+  private val circeYamlV = "0.14.1"
+  private val commonsCodecV = "1.15" // via: https://commons.apache.org/proper/commons-codec/
+  private val commonsCsvV = "1.9.0"
+  private val commonsIoV = "2.11.0" // via: https://commons.apache.org/proper/commons-io/
+  private val commonsLang3V = "3.12.0"
   private val commonsMathV = "3.6.1"
+  private val commonNetV = "3.8.0" // via: https://commons.apache.org/proper/commons-net/
   private val commonsTextV = "1.9"
-  private val configsV = "0.6.0"
+  private val configsV = "0.6.1"
   private val delightRhinoSandboxV = "0.0.15"
+  private val diffsonSprayJsonV = "4.1.1"
   private val ficusV = "1.5.0"
   // The "com.vladsch.flexmark" % "flexmark-profile-pegdown" % flexmarkV dependency is an implicit, version-specific
   // runtime dependency of ScalaTest. At the time of this writing this is the newest version known to work.
   private val flexmarkV = "0.36.8" // scala-steward:off
-  private val fs2V = "2.5.4"
+  private val fs2V = "2.5.9" // scala-steward:off (CROM-6564)
   // Scala Steward opened PR #5775 titled "Update fs2-io from 2.0.1 to 2.4.3" to upgrade the following dependency.
   // However that PR was actually attempting an upgrade from 1.0.5 to 2.4.3 which is a much more significant
   // undertaking, resulting in some thoroughly broken statsd proxy code. As this component lacks tests and is
   // probably not the most important corner of the Cromwell repo, going to punt on this for now.
+  // NOTE: fs2VStatsDProxy usage within allProjectDependencies breaks sbt whatDependsOn.  If you want to use
+  // whatDependsOn temporarily set fs2VStatsDProxy to fs2V and revert back before trying to compile.
   private val fs2VStatsDProxy = "1.0.5" // scala-steward:off
-  private val googleApiClientV = "1.31.3"
-  private val googleCloudBigQueryV = "1.127.11"
-  private val googleCloudKmsV = "v1-rev20210312-1.31.0"
-  private val googleCloudMonitoringV = "2.0.14"
-  private val googleCloudNioV = "0.61.0-alpha" // scala-steward:off
-  private val googleCloudStorageV = "1.113.14"
-  private val googleGaxGrpcV = "1.62.0"
-  private val googleGenomicsServicesV2Alpha1ApiV = "v2alpha1-rev20210322-1.31.0"
+  private val googleApiClientV = "1.32.1"
+  private val googleCloudBigQueryV = "2.1.4"
+  // latest date via: https://mvnrepository.com/artifact/com.google.apis/google-api-services-cloudkms
+  private val googleCloudKmsV = "v1-rev20210812-1.32.1"
+  private val googleCloudMonitoringV = "3.0.2"
```

Comment:
Claims to include a breaking change in 3.x: https://github.com/googleapis/java-monitoring/releases/tag/v3.0.0 but I'm going to guess that it won't affect us.